.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 
     // @typescript-eslint/eslint-plugin
     '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/member-ordering': 'error',
+    '@typescript-eslint/member-ordering': 'off',
```

Comment:
We cna just remove that since the rule isn't part of `@typescript-eslint/recommended`